src/tests/ftest/nvme/enospace.py

```diff
@@ -461,6 +461,7 @@ def run_enospace_foreground(self, log_file):
 
         # Fill 75% of current SCM free space. Aggregation is Enabled so NVMe space will
         # start to fill up.
+        # pylint: disable-next=logging-too-few-args
         self.log.info('--Filling 75% of the current SCM free space--')
```

Comment:
```suggestion
        self.log.info(r'--Filling 75% of the current SCM free space--')
```

Reply:
Sorry, wrong suggestion, try this instead:
```
self.log.info('--Filling 75%% of the current SCM free space--')
```

Reply:
I've tried before and it doesn't work